backend/src/main/java/org/sefglobal/core/controller/CertificateController.java

```diff
@@ -0,0 +1,39 @@
+package org.sefglobal.core.controller;
+
+import org.sefglobal.core.exception.ResourceNotFoundException;
+import org.sefglobal.core.model.Certificate;
+import org.sefglobal.core.service.CertificateService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/fellowship")
```

Comment:
Hi you can add the `/certificates ` mapping to class level

Reply:
Done!